package.json

```diff
@@ -92,6 +92,7 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.25.1",
     "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-plugin-unicorn": "40.1.0",
```

Comment:
This isn't the latest version of eslint-plugin-unicorn, because versions 41 onwards depend on eslint 8+.